pkg/network/route_cache.go

```diff
@@ -48,26 +47,53 @@ type routeCache struct {
 	cache  *lru.Cache
 	router Router
 	ttl    time.Duration
-
-	size    *atomic.Uint64 `stats:""`
-	misses  *atomic.Uint64 `stats:""`
-	lookups *atomic.Uint64 `stats:""`
-	expires *atomic.Uint64 `stats:""`
 }
 
-const defaultTTL = 2 * time.Minute
+const (
+	defaultTTL                       = 2 * time.Minute
+	routeCacheTelemetryModuleName    = "network_tracer_gateway_lookup_route_cache"
+	netlinkRouterTelemetryModuleName = "network_tracer_gateway_lookup_route_cache_router"
```

Comment:
The `netlinkRouterTelemetryModuleName` doesn't really match the actual string. I wouldn't mention netlink the variable name unless it is also in the string.